### ISSUE TITLE
WebSocket error when upgrade to JDK 17

### ISSUE THREAD

**Issue by vellrya** [jooby-project/jooby#2585]
Hello, I am upgarded my JDK version to "OpenJDK Runtime Environment Corretto-17.0.3.6.1" from "Corretto-15.0.2.7.1" and ran into following exception:

```
configurer.onConnect { ws ->
    ws.send(JSONObject().put("connected", true).toString())
}
```

Jooby 2.11.0 (error appeared in log, but message sended):
```
[2022-05-13 19:52:19,277]-[worker I/O-12] ERROR org.xnio.listener - XNIO001007: A channel event listener threw an exception
java.io.IOError: java.io.FileNotFoundException: Invalid file path
	at org.xnio.channels.Channels$4.run(Channels.java:1103)
	at org.xnio.channels.Channels$4.run(Channels.java:1093)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:318)
	at org.xnio.channels.Channels.<clinit>(Channels.java:1093)
	at org.xnio.ChannelListeners$14.handleEvent(ChannelListeners.java:420)
	at org.xnio.ChannelListeners$14.handleEvent(ChannelListeners.java:409)
	at org.xnio.ChannelListeners.invokeChannelListener(ChannelListeners.java:92)
	at org.xnio.ChannelListeners$4.run(ChannelListeners.java:147)
	at org.xnio.nio.WorkerThread.safeRun(WorkerThread.java:612)
	at org.xnio.nio.WorkerThread.run(WorkerThread.java:479)
Caused by: java.io.FileNotFoundException: Invalid file path
	at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:229)
	at java.base/java.io.FileOutputStream.<init>(FileOutputStream.java:123)
	at org.xnio.channels.Channels$4.run(Channels.java:1098)
	... 9 common frames omitted
```

Jooby 2.14.1 (message doesn't send):
```
java.lang.IllegalStateException: Attempt to send a message on closed web socket
	at io.jooby.internal.utow.UtowWebSocket.send(UtowWebSocket.java:116)
	at io.jooby.internal.utow.UtowWebSocket.send(UtowWebSocket.java:100)
	at io.jooby.WebSocket.send(WebSocket.java:164)
	at my.project.App$1$5$1$1.onConnect(App.kt:71)
	at io.jooby.internal.utow.UtowWebSocket.lambda$fireConnect$0(UtowWebSocket.java:175)
	at io.jooby.internal.utow.UtowWebSocket.lambda$webSocketTask$4(UtowWebSocket.java:302)
	at org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
	at org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:1982)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1486)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1348)
	at org.xnio.XnioWorker$WorkerThreadFactory$1$1.run(XnioWorker.java:1282)
	at java.base/java.lang.Thread.run(Thread.java:833)
```

Exception also appears if I try to run a version compiled with <jvmTarget>15</jvmTarget> on JDK 17.
Kotlin version: 1.4.32.

**Comment by jknack** [comment]
Try latest: 2.14.1

**Comment by vellrya** [comment]
> Try latest: 2.14.1

> Jooby 2.14.1 (message doesn't send):

Of course, I tried it. On this version, the message does not send at all (log attached in #1).

**Comment by jknack** [comment]
Sorry, I missed that. Should be a something easy to fix, do you want to give it a try?

**Comment by vellrya** [comment]
Even though I use your library in almost all my projects, my knowledge of its inner workings is not enough to fix the error)

I tried JDK 15 with 2.13.0 and below - no errors.
JDK 15 with 2.14.0 and 2.14.1 - "Attempt to send a message on closed web socket".

**Comment by jknack** [comment]
No worries. Yes, there was an error on undertow which is fixed in 2.14... but seems I introduced a new bug too: "Attempt to send a message on closed web socket" while doing #2210
